mljob/templates/result_base.html

```diff
@@ -50,11 +50,11 @@ <h4 class="col-12 p-2">
                     <br>
                     <div class="row">
                         <div class="col-xs-12 col-sm-12">
-                            <label>The median value of the log2(auPRC/prior)
-                                for 5-fold cross validation is &nbsp</label>{{( avgps )}}.
-                            </label>&nbsp
+                            {# avgps is a list of strings of these values #}
+                            <label>Results for 3-fold cross validation in terms of <em>log2(auPRC/prior)</em>: <strong>{{ avgps|join(", ") }}.</strong>
```

Comment:
We are hard coding the folds here but making it a variable elsewhere (as a function variable in run_SL). This isn't a problem since we aren't giving the ability to change the number of folds so I am approving but it is with commenting here for now